style.css

```diff
@@ -0,0 +1,105 @@
+* {
+    box-sizing: border-box;
+}
+a {
+    color: inherit;
+    text-decoration: inherit;
+}
+body {
+    margin: 0;
+    font-size: 20px;
```

Comment:
apparently not doing anything

Reply:
This refers to the text which the rest of the webpage will inherit.

Reply:
font-size